tests/test_diffeq/test_odefiltsmooth/test_string_to_measmodel.py

```diff
@@ -22,13 +22,15 @@ def ivp():
 @pytest.fixture
 def prior(ivp):
     ode_dim = ivp.dimension
-    prior = statespace.IBM(ordint=2, spatialdim=ode_dim)
+    prior = randprocs.markov.integrator.IntegratedWienerTransition(
+        num_derivatives=2, wiener_process_dimension=ode_dim
+    )
     initrv = randvars.Normal(
         mean=np.zeros(prior.dimension),
         cov=np.eye(prior.dimension),
         cov_cholesky=np.eye(prior.dimension),
     )
-    prior_process = randprocs.MarkovProcess(
+    prior_process = randprocs.markov.MarkovProcess(
         transition=prior, initrv=initrv, initarg=0.0
     )
     return prior_process
```

Comment:
We could just create an `IntegratedWienerProcess` directly here, right?

Reply:
yeah we could. However, it does not matter too much because this whole test suite is not part of the codebase anymore once #486 and #498 are merged.